Scripts/SAPSetupAssistantValidationScripts/SetupAssistantSAPValidateNetworkConnection.ps1

```diff
@@ -0,0 +1,29 @@
+param (
```

Comment:
A comment block explaining each variable would be a nice to have.

```
<#
.SYNOPSIS
    Tests network connection to a specified host and port.
.DESCRIPTION
    This script attempts to establish a TCP connection to the specified host and port,
    and reports the result in a JSON format.
.PARAMETER HostName
    The hostname or IP address to connect to.
.PARAMETER Port
    The port number to connect to.
#>
```

Reply:
Added to all scripts